modules/tinymce/src/core/main/ts/api/Tinymce.ts

```diff
@@ -61,6 +62,44 @@ import VK from './util/VK';
 import XHR from './util/XHR';
 import WindowManager from './WindowManager';
 
+interface DOMUtilsNamespace {
```

Comment:
I'm not sure how I feel about using `Namespace` as part of a name, especially for something that isn't actually a namespace, per the TypeScript definition of it, as far as I can tell.

Reply:
It's the representation of the namespace merged with the function. See: https://github.com/tinymce/tinymce/blob/develop/modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts#L1892